additions/permanent/domains.list

```diff
@@ -2261,3 +2261,23 @@ zh-trust.com
 zk-manta-airdrop.pages.dev
 zt-za.fr
 zttmct-tlemp.web.app
+streettokitchen.com
```

Comment:
Probably al fine, but nothing beats a screenshot, it is the best of both worlds, one of the Right site and one (in this case an example or two) of the phishing site for faster processing.

I'm not approving this, as I can not read up on it all. (Cataracts) but I'll like to say god work, you have indeed all the other data needed for easily find proofs pf concept